### ISSUE TITLE
ETA on when RInno will function again?

### ISSUE THREAD

**Issue by sbr2567** [ficonsulting/RInno#149]
I noticed this package is still listed as active, but the current errors render it unusable. Would be nice to have some kind of update. Is there any further development being made or is the project left on hold at this time.

Cheers.

**Comment by chasemc** [comment]
Agreed, @Dripdrop12.  The last commit to master was on Feb 6, 2019 by myself.  I moved on and wrote my own package to accomplish this but still get emails on the issues here and it seems RInno is unsupported at this time. 

Can you please update the `https://www.repostatus.org/` status in the README?
